pkg/argocd/git.go

```diff
@@ -44,7 +51,16 @@ func TemplateCommitMessage(tpl *template.Template, appName string, changeList []
 	// writer of a template.
 	changes := make([]commitMessageChange, 0)
 	for _, c := range changeList {
-		changes = append(changes, commitMessageChange{c.Image.ImageName, c.OldTag.String(), c.NewTag.String()})
+		newTag := c.NewTag.String()
```

Comment:
wrong place to do this, this was a pure function, keep it as such.

Reply:
The data for template rendering is generated here. We need to modify this function to make the changes.